code/datums/components/electrified_buckle.dm

```diff
@@ -153,9 +153,10 @@
 /datum/component/electrified_buckle/proc/on_update_overlays(atom/movable/source, list/overlays)
 	SIGNAL_HANDLER
 	var/overlay_layer = length(source.buckled_mobs) ? ABOVE_MOB_LAYER : OBJ_LAYER
-	for (var/image/overlay_image in requested_overlays)
-		overlay_image.layer = overlay_layer
-		overlays += overlay_image
+	for (var/mutable_appearance/electrified_overlay as anything in requested_overlays)
```

Comment:
is an as anything here safe? (text?) maybe an if(!(isimage && isma()) guard check?
also TECHNICALLY risks issues with static MAs but like I don't care

Reply:
if you wanna be safe do a copy first